src/main/java/org/generationcp/middleware/service/api/study/StudyInstanceService.java

```diff
@@ -0,0 +1,14 @@
+package org.generationcp.middleware.service.api.study;
+
+import org.generationcp.middleware.pojos.workbench.CropType;
+import org.generationcp.middleware.service.impl.study.StudyInstance;
+
+import java.util.List;
+
+public interface StudyInstanceService {
+
+	StudyInstance createStudyInstance(final CropType crop, final int datasetId, final int instanceNumber);
```

Comment:
As commented in BMSAPI, we should not expose instanceNumber as parameter. By default, the instance number is incremented from the last existing instance

Reply:
Done.